### PR TITLE
Enable CI

### PR DESCRIPTION
**Pull request by bcomnes**
Looks like we got actions enabled!

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -20,6 +19,7 @@ test('const chrome = nanochrome(url[, opts])', (t) => {
       t.notOk(err)
       chrome.close((err) => {
         t.notOk(err)
+        server.close()
```
Comment (bcomnes) [inline test.js]: Moving this to here closes the server reliably, however, this does seem to indicate our chrome process is never quite making a connection to our server.  Problem with the test?

Comment (jwerle) [reply test.js]: maybe connecting to localhost in the CI server is an issue for headless chrome?

Comment (bcomnes) [reply test.js]: It was hanging locally on my system too, as if the route was never firing.

Comment (jwerle) [reply test.js]: ah i think we need to listen on `host` in the `listen()` call

**Comment by bcomnes** [comment]
Eureka !  Maybe we can wrap some of the xfvb stuff into its own action.

**Pull request merged**